platforms/ide/ide-native/src/main/java/org/gradle/ide/visualstudio/internal/AbstractCppBinaryVisualStudioTargetBinary.java

```diff
@@ -59,7 +58,6 @@ public LanguageStandard getLanguageStandard() {
         return LanguageStandard.from(getBinary().getCompileTask().get().getCompilerArgs().get());
     }
 
-    @Internal
```

Comment:
This type is part of the mutable model and is no longer used (or should be used) as a @Nested bean.